src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,72 @@
+package seedu.addressbook.ui;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.commands.CommandResult;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import static seedu.addressbook.common.Messages.MESSAGE_USING_STORAGE_FILE;
+
+/**
+ * Format text (eg. decorations) for display
+ */
+public class Formatter {
+
+    /** A decorative prefix added to the beginning of lines printed by AddressBook */
+    private static final String LINE_PREFIX = "|| ";
+
+    /** A platform independent line separator. */
+    private static final String LS = System.lineSeparator();
+
+    private static final String DIVIDER = "===================================================";
+
+
+    /**
```

Comment:
Good, this is how it should appear. `@param` for every single parameter and `@return` for every method that returns a value